src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOQuerySupport.scala

```diff
@@ -233,7 +233,7 @@ trait ElasticSearchDAOQuerySupport extends ElasticSearchDAOSupport with ElasticS
     results map { suggestResult =>
       val compSugg: CompletionSuggestion = suggestResult.getSuggest.getSuggestion(suggestionName)
       val options : Seq[CompletionSuggestion.Entry.Option] = compSugg.getEntries.get(0).getOptions.asScala
```

Comment:
Should we guard against and/or handle `compSugg.getEntries` returning an empty `List`? I couldn't see a test for it, either.

Reply:
I doubt it would get that far, because there are a couple ways for `getSuggestion()` to return `null` first.
https://github.com/elastic/elasticsearch/blob/093218e052acc84338e90e1ab77ed413843a6244/core/src/main/java/org/elasticsearch/search/suggest/Suggest.java#L106

We also have a top-level exception handler, though it's specific to `FireCloudException`s
https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryService.scala#L242

It could just be that this case is undefined behavior where our system is concerned, and we don't care what happens ("Don't do that.").

Reply:
I've added some null/empty protection for the sake of safety, though it does make the code more verbose.

Reply:
Doesn't seem too necessary to me (would this ever actually happen?), but appears correct